### DISCUSSION TITLE
Timer before CDN requests?

### DISCUSSION THREAD

**Discussion by furu00** [Tyrrrz/DiscordChatExporter#858] [Q&A] [answered]
Hi,

Is it Possible to have a wait timer before every cdn request?
I see other apps do this to prevent a ban.
Maybe someone can point me to the part of the script where I can just insert a "wait xxx"?

Tahnks.

**Comment by Technetium1** [comment, marked as answer]
https://github.com/Tyrrrz/DiscordChatExporter/blob/8d4e6b77c4e9e97fca3d9615206e11d72f6eff60/DiscordChatExporter.Core/Utils/Http.cs#L32-L58
